pages/docs/custom-dashboards.mdx

```diff
@@ -0,0 +1,65 @@
+---
+description: Create self-service dashboards to visualize, monitor, and share insights from your LLM application data.
+---
+
+# Custom Dashboards
+
+<AvailabilityBanner
+  availability={{
+    hobby: "public-beta",
+    core: "public-beta",
+    pro: "public-beta",
+    enterprise: "public-beta",
+    selfHosted: "not-available",
+  }}
+/>
+
+Langfuse custom dashboards allow you to create personalized views of your LLM application metrics.
+Build widgets that visualize specific aspects of your data and arrange them into comprehensive dashboards
+to monitor performance, usage patterns, and other important metrics.
+
+## Dashboard Widgets
+
+Widgets are individual visualization components that display specific metrics.
+Start your dashboard journey by creating your first widget.
+
+### Adding a Widget
+
+1. Navigate to the `Dashboards` navigation item and select the `Widgets` tab.
+2. Click **+ New Widget**
+3. Configure your data source and display settings
+4. Click **Save** to store the Widget.
+
+{/* SCREENSHOT: Show the widget creation interface with options for selecting metrics and configuring the display */}
+
+## Creating a Dashboard
+
+Creating a dashboard is a straightforward process:
+
+1. Navigate to the **Dashboards** tab in your Langfuse project
+2. Click **+ New Dashboard**
+3. Give your dashboard a descriptive name
+4. Start adding widgets to build your customized view
+
+{/* SCREENSHOT: Show the dashboard creation interface with the New Dashboard button and empty dashboard ready for widgets */}
+
+## Sharing and Collaboration
+
+Dashboards are shared across your project, allowing team members to:
+
+- View the same dashboard with real-time data
+- Create specialized dashboards for different teams or use cases
+- Reference dashboards in discussions and documentation
+
+## Use Cases
+
+- **Performance Monitoring**: Track response times, token usage, and costs
+- **Quality Assessment**: Monitor evaluation scores and user feedback
+- **Usage Patterns**: Analyze user interactions and session metrics
+- **Model Comparison**: Compare performance across different models or prompts
+
```

Comment:
We could add a roadmap header here with planned changes

- templates
- customize the home dashboard
- query api to get the metrics directly